.env.example

```
@@ -22,22 +22,23 @@ JWT_AUDIENCE=Web
 JWT_LIFETIME=30
 JWT_KEY=NdYrGgZpfEWHD86idwwBFrQlf95wIK0VJwxbHgS0i9M
 
-# RefreshToken
+# Refresh Token
 RT_LIFETIME=15
 
+# Admin Seeder
+ADMIN__FULLNAME=Omar Goher
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)